internal/config/config.go

```diff
@@ -49,9 +50,11 @@ func New() *Config {
 // GlobalFlags holds information on the flags shared among all commands
 type GlobalFlags struct {
 	ConfigFile    string        `valid:"-"`
-	OutputFormat  string        `valid:"in(human|json|yaml)"`
-	Colors        bool          `valid:"-"`
 	ClientTimeout time.Duration `valid:"-"`
+	Colors        bool          `valid:"-"`
+	Debug         bool          `valid:"-"`
+	OutputFormat  string        `valid:"in(human|json|yaml)"`
+	Wait          bool          `valid:"-"`
```

Comment:
This might've beeb added with merge. Is it wrongly here?

Reply:
Wait at least has been removed at some point, otherwise seems fine?